### PR TITLE
Allow importing from a url

### PR DESCRIPTION
**Pull request by karissa**
now you can `dat import` from a URL:

example:
```
dat import http://samplecsvs.s3.amazonaws.com/Sacramentorealestatetransactions.csv -d sacrealty
```

thoughts?

### CODE REVIEW

File: bin/import.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 var fs = require('fs')
 var pump = require('pump')
+var url = require('url')
```
Comment (maxogden) [inline bin/import.js]: minor nit, but I try and put all core requires at the top (so this should go below `fs`)

File: package.json [hunk 2]
```diff
@@ -45,6 +45,8 @@
     "pump": "^1.0.0",
     "pumpify": "^1.3.3",
     "relative-date": "^1.1.2",
+    "request": "^2.58.0",
```
Comment (maxogden) [inline package.json]: request is quite a heavy dependency... I think we should try using `require('got')` instead

**Comment by maxogden** [comment]
+1 to merging

File: bin/import.js [hunk 3]
```diff
@@ -55,14 +57,30 @@ function handleImport (args) {
...
+  function getReadStream (location, cb) {
+    fs.exists(location, function (exists) {
+      if (exists) {
+        cb(null, fs.createReadStream(location))
+      } else {
+        var u = url.parse(location)
+        if (u.protocol && u.protocol.match(/http/)) {
+          cb(null, got(location))
+        }
+        else cb(new Error('File location does not exist.'))
```
Comment (mafintosh) [inline bin/import.js]: i think standard will complain about this style

Comment (mafintosh) [inline bin/import.js]: in general `return cb(...)` can solve most of these nested ifs :)

Comment (karissa) [inline bin/import.js]: huh weird, standard didn't complain (we have commit hooks now)

but i fixed the style

Comment (maxogden) [inline bin/import.js]: i think the style he was referring to was the mixed brackets

```
if (foo) { baz() }
else bar()
```

usually its all one or the other, e.g.

```
if (foo) { baz() }
else { bar () }

if (foo) baz()
else bar()
```

but standard doesnt seem to care

**Pull request merged**